types/json/path_expr.go

```diff
@@ -15,13 +15,13 @@
 package json
```

Comment:
Can you enable nogo for ```types/json``` ?

Reply:
how can I do that?

Reply:
You can edit ```./build/nogo_config.json``` and add your edited file path into ```only_files```. [Here](https://github.com/pingcap/tidb/pull/37070) is the example.

Reply:
I find this pr will be cherrypicked. so you can linter this code next time.

Reply:
it will not be cherry-picked, I cannot remove the label